src/ui/components/component.js

```diff
@@ -22,7 +22,13 @@ export default class Component {
     this.moduleId = null;
 
     /**
-     * Unique name of this component instance
+     * UUID for the component.
+     * @type {number}
```

Comment:
I think uuid's are normally generated strings but I'm not sure if that's necessary here? javascript's MAX_SAFE_INTEGER  is 9007199254740991 and I don't think we'll ever hit that cap